### ISSUE TITLE
API - Read-only  mode

### ISSUE THREAD

**Issue by gingervik** [BeamMW/beam#489]
Please, add ability to use API wallet in read-only mode

**Comment by nesbox** [comment]
read-only mode implemented with #488
the following api methods now have access rights (only if you enabled ACL in the API)
```
create_address      - write access
validate_address    - read access
tx_send             - write access
tx_status           - read access
tx_split            - write access
tx_cancel           - write access
get_utxo            - read access
tx_list             - read access
wallet_status       - read access
```
more info here https://github.com/BeamMW/beam/issues/488#issuecomment-467508372

**Comment by Sergei-Beam** [comment]
Checked
